libcaf_test/caf/test/reporter.cpp

```diff
@@ -380,6 +392,10 @@ class default_reporter : public reporter {
     level_ = level;
   }
 
+  void disable_colors(bool plain_output) override {
+    plain_output_ = plain_output;
```

Comment:
This is a setter, so the associated member should have the same name with a `_` suffix according to our [style guide](https://github.com/actor-framework/actor-framework/blob/master/CONTRIBUTING.md#naming):

```c++
  void disable_colors(bool new_value) override {
    disable_colors_ = new_value;
  }
```

or:

```c++
  void plain_output(bool new_value) override {
    plain_output_ = new_value;
  }
```

Given that this gets filled from a config option named `no-colors`, the most logical choice would actually probably be:

```c++
  void no_colors(bool new_value) override {
    no_colors_ = new_value;
  }
```